frontend/src/metabase/visualizations/echarts/cartesian/scatter/series.ts

```diff
@@ -6,45 +6,48 @@ import type { RenderingContext } from "metabase/visualizations/types";
 import type {
   DataKey,
   Datum,
+  Extent,
   GroupedDataset,
   SeriesModel,
 } from "../model/types";
 
 const BUBBLE_SCALE_FACTOR_MAX = 64;
 
+const MIN_BUBBLE_SIZE = 14;
+
 // TODO: refine the scaling curve when implementing the dynamic scatter plot
```

Comment:
![Screenshot 2024-01-01 at 3.43.57 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/609a53d0-af4b-4869-8d48-ec57799712a0.png)

I don't think it makes sense to use the old scaling curve from the dc.js implementation, it doesn't look good in app and especially not in static viz, which is why I didn't use it initially and why I left this comment here.

Can we either fix it in this PR, or before we finish the project? I've filed an issue for it: https://github.com/metabase/metabase/issues/37183

Reply:
Yup, the existing logic looks bad in many cases because we always anchored domain to start with zero. I will fix it here https://github.com/metabase/metabase/pull/37196

Reply:
Thanks!